### DISCUSSION TITLE
Error al querer guardar las opciones

### DISCUSSION THREAD

**Discussion by Jermaine316** [bucanero/pkgi-ps3#41] [Q&A] [answered]
Hola, tengo un pequeño error, cuando quiere darle a la opción de backgraund no se guarda y me pone la opción de direc. Tl. Alguien sabe cómo solucionar este problema?

**Comment by Elmatecocido** [comment, marked as answer]
Para salir del menú después de cambiar las opciones, no le des al círculo, sino al triángulo, porque sino no te guarda las cosas que cambiaste

**Comment by Jermaine316** [comment]
Ese era mi error😅 muchas gracias por la ayuda! 😀
